src/components/ChatLog.jsx

```diff
@@ -0,0 +1,42 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry.jsx';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+  const chatMessages = props.entries.map((message) => {
+    return (
+      <li key={message.id}>
```

Comment:
Nice use of the `message.id` for list item `key` values since our messages have unique ids!